src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1693,6 +1693,13 @@ public override BoundNode VisitFieldAccess(BoundFieldAccess node)
 
         private void VisitFieldAccessInternal(BoundExpression receiverOpt, FieldSymbol fieldSymbol)
         {
+            if (receiverOpt == null)
+            {
+                // VisitLvalue and VisitRValue don't accept null arguments, and if receiverOpt is null
+                // then there's nothing left to visit anyway.
```

Comment:
```
    TypeName.fieldName // receiverOpt is TypName
vs 
    fieldName // receiverOpt is null  
```

This has been working fine for us so far because non-fixed fields are treated as Rvalues, while fixed fields go through the lvalue path (while binding nameof)